airbyte-commons/src/main/java/io/airbyte/commons/features/EnvVariableFeatureFlags.java

```diff
@@ -64,6 +68,21 @@ public String fieldSelectionWorkspaces() {
     return getEnvOrDefault(FIELD_SELECTION_WORKSPACES, "", (arg) -> arg);
   }
 
+  @Override
+  public String strictComparisonNormalizationWorkspaces() {
+    return getEnvOrDefault(STRICT_COMPARISON_NORMALIZATION_WORKSPACES, "", (arg) -> arg);
+  }
+
+  @Override
+  public String strictComparisonNormalizationTag() {
+    return getEnvOrDefault(STRICT_COMPARISON_NORMALIZATION_TAG, "strict_comparison", (arg) -> arg);
+  }
+
+  @Override
+  public String nonStrictComparisonNormalizationTag() {
```

Comment:
Should be the last comment about env vars, I think this (+ the chain of constants and the env propagation) is no longer needed.